questionary/prompts/common.py

```diff
@@ -343,9 +347,19 @@ def _init_choices(
 
             self.choices.append(choice)
 
+    @property
+    def filtered_choices(self):
+        if not self.search_filter:
+            return self.choices
+        filtered = [
+            c for c in self.choices if self.search_filter.lower() in c.title.lower()
```

Comment:
I'm ok with this search mechanism, but it is not a prefix search like #42. Does this matter? Are there any downsides to searching for any occurrence of the substring? I suppose in the future, we can make the filter customisable, so it doesn't matter too much...

Reply:
After using the prefix search for a few days it felt unnatural and not intuitive. I personally don't know any application that prefers a prefix search above a substring search. 

A fuzzy search would probably be the best, but add unnecessary complexity and/or dependencies to this library.

Reply:
Thanks for explaining